apps/schema/static/schema/components/shipments/schema.yaml

```diff
@@ -161,11 +165,13 @@ createAttributes:
   - $ref: '#/shipment/storageCredentialsId'
   - $ref: '#/shipment/ownerId'
   - $ref: '#/shipment/vaultId'
+  - $ref: '#/shipment/vaultUri'
   - $ref: '#/shipment/shipperWalletId'
   - $ref: '#/shipment/carrierWalletId'
-  - $ref: '#/shipment/deviceId'
```

Comment:
I know I said to remove `device` from the included resources in our responses, which I think is still a good decision, but can we keep device_id as an attribute that is returned? We still need to be able to see the ID of the device that's associated with a shipment, even if the Shipment isn't in 'included'. (Or maybe device_id should be in the 'relationships' section of the response?)

Reply:
Yeah, we we moved it from the included to the "relationships" section